front/src/utils/config.js

```diff
@@ -1,6 +1,6 @@
 const config = {
   api: {
-    url: process.env.API_URL
+    url: typeof window === "undefined" ? process.env.API_URL : process.env.NEXT_PUBLIC_API_URL
```

Comment:
Does this variables have the same values? If yes why wouldn't you use only one ?

Reply:
It's not the same value

Reply:
I wrote a comment on the file to not forgot what its used for